Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/EditorVertexSelection.cpp

```diff
@@ -1098,6 +1238,21 @@ namespace AzToolsFramework
             });
     }
 
+    template<typename Vertex>
+    EditorVertexSelectionVariable<Vertex>::EditorVertexSelectionVariable(const AZ::EntityComponentIdPair& entityComponentIdPair)
+    {
+        EditorVertexSelectionVariableRequestBus::Handler::BusConnect(entityComponentIdPair);
+    }
+
+    template<typename Vertex>
+    EditorVertexSelectionVariable<Vertex>::~EditorVertexSelectionVariable()
+    {
+        if (EditorVertexSelectionVariableRequestBus::Handler::BusIsConnected())
+        {
+            EditorVertexSelectionVariableRequestBus::Handler::BusDisconnect();
+        }
```

Comment:
Nit: The `BusDisconnect` method already checks if its connected first, so I think you can skip that check here:

```
if (BusIsConnected())
{
    BusType::DisconnectInternal(*context, m_node);
}
```

Reply:
Got it, I will honestly move on with it if it passes AR, but will keep this in mind for a follow-up!